services/monitoring/docker-compose.yml

```diff
@@ -196,6 +196,8 @@ services:
           memory: 64M
 
   grafana:
+    dns:
+      8.8.8.8
```

Comment:
This dns entry here. why is that?
I've seen you have that in the deployer docker-compose.aws.yml and here it is in the base docker-compose. any reason?

Reply:
it looks like it is only in grafana, adminer, deployer... why not graylog?? is this really necessary? or are these remainders of tests? if so please remove them.

Reply:
This a problem that I met several times within AWS, some container doesn't access internet if I don't set the DNS myself. True that I should that graylog could have it also one day. I will add it

Reply:
but how come it happens only sometimes? it must be something else don't you think?

Reply:
I didn't find the cause. I was thinking first it would depend of the node it is deployed but not at all. Sometimes, just restarting it it would work and some other times no. I believe this is because the DNS from the base DNS (from ubuntu) are not correctly copied into the container. But I would need to take a deeper look to understand why it happens only sometimes..

Reply:
well if you are sure this does not have ANY side effect you can probably set it everywhere. but no one will understand if it is only in some case there. Also, if you do so, add a note next to it. Or better find out why it is needed, and if it is not, remove it.